### PR TITLE
Show unformatted rankings on UserProfile

### PR DESCRIPTION
**Pull request by koalarick**
Sorry as I work through shared files, Prettier changes up some formatting.

### CODE REVIEW

File: client/components/users/UserRankings.jsx [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-import React from "react";
+import { getOverallRankings } from "../../../api_master";
+
+function UserRankings({ loggedInUser }) {
+  const [rankings, setRankings] = useState({});
+
+  var refreshRankings = () => {
+    if (loggedInUser.id) {
+      getOverallRankings(loggedInUser.id).then((res) => {
+        setRankings(res.data);
```
Comment (john-cooke832) [inline client/components/users/UserRankings.jsx:11]: Not a big deal, but in past code reviews with Kim and Tom, they said it is better to give this res object a more explicit name, in this case, I'm guessing something like rankings.

File: helperFunctions.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  from
+    (select u.id, u.username, count(q.id) as total,
+    ${rankingType} () over (order by count(q.id) desc) rank
+    from ${eligibleUsers}
+    left join user_completed_quizzes q
+    on q.id_users = u.id
+    -- could add quiz id criteria here
+    group by 1,2) rankings
+  where id = ${userId}
+  group by 1,2`;
```
Comment (john-cooke832) [inline helperFunctions.js:28]: Not familiar enough with how this should look, and I feel bad about it. If it works, though, 👍

File: client/components/users/UserRankings.jsx (hunk 1 above)
Comment (koalarick) [reply client/components/users/UserRankings.jsx]: Addressed

File: server/index.js [hunk 3]
```diff
@@ -478,6 +478,46 @@ app.get("/users", async (req, res) => {
...
+  let promises = [];
+  let results = {};
+  let userId = req.params.userId;
+
+  try {
+    promises.push(
+      pool
+        .query(rankingQueryMaker(userId, "friends", "percent_rank"))
+        .then((res) => {
+          results.friendPercentile = res.rows[0].rank;
```
Comment (john-cooke832) [inline server/index.js:494]: Again, res

File: server/index.js [hunk 4]
```diff
@@ -478,6 +478,46 @@ app.get("/users", async (req, res) => {
...
+        })
+    );
+
+    promises.push(
+      pool.query(rankingQueryMaker(userId, "friends", "rank")).then((res) => {
+        results.friendRank = res.rows[0].rank;
```
Comment (john-cooke832) [inline server/index.js:500]: res

File: server/index.js [hunk 5]
```diff
@@ -478,6 +478,46 @@ app.get("/users", async (req, res) => {
...
+      })
+    );
+
+    promises.push(
+      pool
+        .query(rankingQueryMaker(userId, "global", "percent_rank"))
+        .then((res) => {
+          results.globalPercentile = res.rows[0].rank;
```
Comment (john-cooke832) [inline server/index.js:508]: res

File: server/index.js [hunk 6]
```diff
@@ -478,6 +478,46 @@ app.get("/users", async (req, res) => {
...
+        })
+    );
+
+    promises.push(
+      pool.query(rankingQueryMaker(userId, "global", "rank")).then((res) => {
+        results.globalRank = res.rows[0].rank;
```
Comment (john-cooke832) [inline server/index.js:514]: res

File: server/index.js [hunk 7]
```diff
@@ -478,6 +478,46 @@ app.get("/users", async (req, res) => {
...
+      })
+    );
+
+    Promise.all(promises).then(() => res.status(200).send(results));
```
Comment (john-cooke832) [inline server/index.js:518]: status 200 not needed

**Review by john-cooke832 — CHANGES_REQUESTED** [review]
Other than your proclivity for double quotes, there were just a couple of small changes I'd like to see. I don't feel any need to re-review though, so if you want to make the changes, you're good to merge. Great stuff!!

File: server/index.js [hunk 8]
```diff
@@ -29,22 +31,22 @@ app.get('/login', async (req, res) => {
         res.sendStatus(500);
       }
       if (result) {
-        res.status(200).send({user: user});
+        res.status(200).send({ user: user });
```
[the remark this answers is not in the document]
Comment (koalarick) [reply server/index.js:34]: Not my code, so I'm gonna let someone else handle

**Pull request merged**